userspace/falco/configuration.cpp

```diff
@@ -63,6 +63,7 @@ falco_configuration::falco_configuration():
 	m_syscall_evt_drop_max_burst(1),
 	m_syscall_evt_simulate_drops(false),
 	m_syscall_evt_timeout_max_consecutives(1000),
+	m_state_engine_thread_table_size(262144),
```

Comment:
Can we have a macro for the default value? Given it is a "weird" number used twice :)

Reply:
Agreed, funny enough if you search that number in the Linux kernel it is not so odd.

Before I re-push, was wondering about the Windows build failure. is there something in libs that needs to be adjusted?

Reply:
> was wondering about the Windows build failure. is there something in libs that needs to be adjusted?

Don't really know why it is complaining :/ 
```
jsoncpp.lib(json_value.obj) : error LNK2019: unresolved external symbol __imp_modf referenced in function "public: bool __cdecl Json::Value::isInt64(void)const " (?isInt64@Value@Json@@QEBA_NXZ) [D:\a\falco\falco\build\libsinsp\examples\sinsp-example.vcxproj]
jsoncpp.lib(json_writer.obj) : error LNK2019: unresolved external symbol __imp__finite referenced in function "class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl Json::`anonymous namespace'::valueToString(double,bool,unsigned int,enum Json::PrecisionType)" (?valueToString@?A0x7fc2ca38@Json@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@N_NIW4PrecisionType@2@@Z) [D:\a\falco\falco\build\libsinsp\examples\sinsp-example.vcxproj]
jsoncpp.lib(json_writer.obj) : error LNK2019: unresolved external symbol __imp__isnan referenced in function "class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl Json::`anonymous namespace'::valueToString(double,bool,unsigned int,enum Json::PrecisionType)" (?valueToString@?A0x7fc2ca38@Json@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@N_NIW4PrecisionType@2@@Z) [D:\a\falco\falco\build\libsinsp\examples\sinsp-example.vcxproj]
```

Seems a libs issue on jsoncpp; perhaps triggered by: https://github.com/falcosecurity/libs/commit/a89fd5d510fac5b5f2418f263f8d2c4ec80410ef

cc @geraldcombs can you take a look? :pray:  thanks!!

Reply:
Perhaps you could write it as `0x40000` so it's easier to understand. Not sure if you can do the same in yaml as well.

Reply:
> Seems a libs issue on jsoncpp

It might be because libs is linking with the dynamic CRT (MultiThreadedDLL) instead of the static one (MultiThreaded). If that's the case we should be able to fix it here by passing `-DCMAKE_MSVC_RUNTIME_LIBRARY=MultiThreaded` to CMake in the `build-win32-package` job, similar to the `build-libs-others-amd64` job in libs.

Alternatively we could try to fix it in libs, e.g. by trying to override CMAKE_MSVC_RUNTIME_LIBRARY in CMakeLists.txt.

Reply:
@LucaGuerra internally yes we can default to 0x40000 in the macro, but don't you think for adopters that is even more weird than a normal number? 🙃

Reply:
> @geraldcombs whichever you think is the best approach. I can bump libs here to the new latest after we address it in libs if we choose that path.

I opened a libs PR (1695) to link to the static CRT by default. That seemed to make the most sense, since the project favors static builds.